packages/jest-runner/src/index.ts

```diff
@@ -91,17 +87,29 @@ class TestRunner {
                 throw new CancelRun();
               }
 
-              await onStart(test);
+              const sendMessageToJest = (
+                eventName: string,
+                args: Array<unknown>,
+              ) => {
+                this.eventEmitter.emit(eventName, deepCyclicCopy(args));
```

Comment:
```suggestion
                this.eventEmitter.emit(eventName, deepCyclicCopy(args, {keepPrototype: false}));
```

it's the default, but let's be explicit